src/Http/Http.Extensions/gen/RequestDelegateGenerator.cs

```diff
@@ -161,15 +163,22 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                     {
                         codeWriter.WriteLine($"{SymbolDisplay.FormatLiteral("{*path:nonfile}", true)},");
                     }
+
+                    (var verbSymbol, var verbHttp) = endpoint.EmitVerb();
```

Comment:
Instead of returning a tuple, we can have `EmitVerb` set state in the `Endpoint.EmitterContext` to indicate which verbs need to have fields emitted for. We can process that state in a new `helperFields` pipeline. That should hopefully flow with our current pattern and make it easy to rebase on the interceptors branch.